htdocs/asset/class/asset.class.php

```diff
@@ -157,6 +157,11 @@ class Asset extends CommonObject
 	public $status;
 	public $user_cloture_id;
 
+	/**
+	 * @var Asset object oldcopy
+	 */
```

Comment:
The $oldcopy is not already into rhe CommonObject class ?

Reply:
yes but as CommonObject

Reply:
![image](https://user-images.githubusercontent.com/3624836/218562522-c7a6e72f-a069-4d3c-8777-660cb5321c6e.png)
these properties exist on Asset but not on CommonObject